dotcom-rendering/src/web/components/MostViewedFooter.test.tsx

```diff
@@ -87,26 +87,6 @@ describe('MostViewedFooterData', () => {
 		expect(getByTestId(secondHeading)).toHaveStyle(HIDDEN);
 	});
 
-	it('should still show the tab menu when there is only one group of tabs', async () => {
-		useApi.mockReturnValue({ data: responseWithOneTab });
-
-		const { queryByText } = render(
-			<MostViewedFooterData
-				sectionName="Section Name"
-				format={{
-					display: ArticleDisplay.Standard,
-					design: ArticleDesign.Standard,
-					theme: ArticlePillar.News,
-				}}
-				ajaxUrl="https://api.nextgen.guardianapps.co.uk"
-			/>,
-		);
-
-		expect(
-			queryByText(responseWithOneTab.tabs[0].heading),
-		).toBeInTheDocument();
-	});
-
```

Comment:
Is there still a test for not a single tab?

Reply:
I'm not sure I understand what you mean?

Reply:
If the data comes in as a single tab, should we have a test confirming that the header is NOT in the document?

Reply:
Ah commit 97da50bc39439bb1e89d3b25fdd92345073aad82 explains it well!